iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/ratis/RatisClient.java

```diff
@@ -167,4 +210,48 @@ public Action handleAttemptFailure(Event event) {
       return defaultPolicy.handleAttemptFailure(event);
     }
   }
+
+  // This policy is used to raft configuration change
+  private static class RatisEndlessRetryPolicy implements RetryPolicy {
+
+    private static final Logger logger = LoggerFactory.getLogger(RatisEndlessRetryPolicy.class);
+    private static final RetryPolicy defaultPolicy;
+
+    static {
+      String str = "";
+      // 50, 500ms, 40, 1000ms, 30, 1500ms, 20, 2000ms, 10, 2500ms
```

Comment:
What is this comment's meaning? Describe your algorithm more clearly.

And, why define an algorithm in static chunk?

Reply:
it's a pairArray, pair such as (n, t),    use the i-th pair, then client will retry n times, each time sleep t  ms.  

I cancat these pair as str, The MultipleLinearRandomRetry will parse it.

I want to calculate only once,  so I think static chunk will only be executed once

Reply:
I think just using "50, 500ms, 40, 1000ms, 30, 1500ms, 20, 2000ms, 10, 2500ms" will be more clear than building a String by loop.

But, this strategy seems not actually **endless** ?

Reply:
yes, the retry time should give a large number.
if the number is enough large, then the loop is ok